sympy/parsing/maple.py

```diff
@@ -0,0 +1,372 @@
+from typing import Any, Dict, Tuple
+from itertools import product
+import re
+from sympy import sympify
+
+
+def maple(s, additional_translations=None):
+    parser = MapleParser(additional_translations)
+    return sympify(parser.parse(s))
+
+def _deco(cls):
+    cls._initialize_class()
+    return cls
+
+@_deco
+class MapleParser:
+    '''
+    An instance of this class converts a string of a basic Maple
+    expression to SymPy style. Output is string type.
+
+    Examples
+    ========
+
+    >>> from sympy.parsing.maple import maple
+    >>> maple("diff(y(x) ,x) + x")
+    x + Derivative(y(x), x)
+
+    >>> maple("x**2*diff(y(x), x, x) + sqrt(x)")
+    sqrt(x) + x**2*Derivative(y(x), (x, 2))
+
+    >>> maple("(y(x)*sqrt(y(x)^2+x^2)+(y(x)^2-x^2) \
+        *sin(alpha)-2*x*y(x)*cos(alpha))*diff(y(x), \
+        x)+x*sqrt(y(x)^2+x^2)+2*x*y(x)*sin(alpha)+ \
+        (y(x)^2-x^2)*cos(alpha)")
+    x*sqrt(x**2 + y(x)**2) + 2*x*y(x)*sin(alpha) + (-x**2 + y(x)**2)*cos(alpha) + (-2*x*y(x)*cos(alpha) + (-x**2 + y(x)**2)*sin(alpha) + sqrt(x**2 + y(x)**2)*y(x))*Derivative(y(x), x)
```

Comment:
What value are these examples providing?

Reply:
This parser is being developed mainly to port ODEs from Maple for my GSoC project

Reply:
The question was about the value of the example. Keep the docstring simple; put the tests in the test suite. Nothing more than a show of how to call the function is probably necessary in the docstring.